core-plugins/src/e2e-test/resources/errorMessage.properties

```diff
@@ -29,3 +29,6 @@ errorLogsMessageDistinctInvalidFields=Spark program 'phase-1' failed with error:
 errorLogsMessageDistinctInvalidNumberOfPartitions=Spark program 'phase-1' failed with error: Unable to create config \
   for batchaggregator Distinct 'numPartitions' is invalid: Value of field class io.cdap.plugin.\
   batch.aggregator.AggregatorConfig.numPartitions is expected to be a number.. Please check the system logs for more details.
+errorLogsMessageGroupByInvalidFields=Spark program 'phase-1' failed with error: null.
```

Comment:
This error message signifies nothing and does not help in any kind of validation.

Reply:
There is no specific error shown in logs when providing invalid field/aggregates value set as macro. 
<img width="1765" alt="Screenshot 2024-12-10 at 11 30 17 AM" src="https://github.com/user-attachments/assets/d4b0d36e-6514-4451-bf05-9977d3cb7049">
<img width="1765" alt="Screenshot 2024-12-10 at 11 28 09 AM" src="https://github.com/user-attachments/assets/28b9edf3-e13f-4d72-943b-2d9fa8f10485">

Reply:
Created a JIRA: https://cdap.atlassian.net/browse/PLUGIN-1828 to provide proper error message.

Reply:
Okay Thanks!